### ISSUE TITLE
Cant Connect to Basic RDP Hosts

### ISSUE THREAD

**Issue by Heggeg** [boschkundendienst/guacamole-docker-compose#24]
Hello,

first of all thank you for creating the compose script. 

I have deployed the containers exactly as described here. 

Unfortunately I cannot connect to a basic RDP host. 

I am getting desperate.

The target host is a current domain Windows Server 2019. 

- NLA was turned off and on as a test.
- The Ignore certificate checkbox is set.
- Credentials were entered in a wide variety of notations. 

However, the error pattern and the logs of the guacd and guacamole container have not changed. 

Do I have to do anything else to connect to a RDP host with this construct?

[_guacamole_compose_logs.txt](https://github.com/boschkundendienst/guacamole-docker-compose/files/7569495/_guacamole_compose_logs.txt)
[_postgres_guacamole_compose_logs.txt](https://github.com/boschkundendienst/guacamole-docker-compose/files/7569496/_postgres_guacamole_compose_logs.txt)
[_guacd_compose_logs.txt](https://github.com/boschkundendienst/guacamole-docker-compose/files/7569499/_guacd_compose_logs.txt)

**Comment by boschkundendienst** [comment]
Hi,
I am currently not using Guacamole myself but I did a quicktest by cloning my repo and quickly connected to a Windows 7 Workstation via RDP using Guacamole and it worked.

In one of your logs there is clearly stated:

```
guacd[674]: INFO:	Security mode: NLA
``` 

But afaik there is no need to disable NLA on the machine you want to connect to, you just have to set the Security Mode to `any`.

I would take a look at the `CONCURRENCY LIMITS` and set the `Maximum number of connections` and `Maximum number of connections per user` to e.g. 10 and retry.

Also, just for sure you should also verify the connection using a Windows machine and using mstsc.exe and try.

From my point of view the problem either lies in Guacamole itself or your setup so I would advise you to ask for help at the Guacamole Mailing List.

**Comment by Heggeg** [comment]
Hi, 

Yes, I have also blamed it on NLA. 

However, I tried the following discontinued Guacamole container after trying all the latest Docker compose files from Github & WWW.  

With this container I was able to get to the login screen of one of my servers without any problems:
https://hub.docker.com/r/oznu/guacamole

NLA I turned off and tried in connection with your repo, unfortunately without success. 

CONCURRENCY LIMITS I will have a look at and adjust. 

I would like to have Guacamole actually only quite bluntly over port 8080 attainable around then with an already existing Nginx Proxy manager me around HTTPs to worry. 

From normal domain machines a connection via mtsc.exe is possible without problems. 

The Guacamole Mailing List is not yet known to me, I must also look at.
